README.md

```diff
@@ -13,7 +13,7 @@ Include = /etc/pacman.d/mirrorlist
 ```
 
 Then install the following dependencies:  
-`sudo pacman -S --needed python xdelta3 jre8-openjdk lib32-libpulse`  
+`sudo pacman -S --needed xdelta3 jre8-openjdk lib32-libpulse`
```

Comment:
You're using `sed` as well, which might be preinstalled on some distros but it is not apart of coreutils. It should be documented as a dependency I think. On NixOS we use the `gnused` package.

Reply:
Oh, thought both unzip and sed were part of coreutils
I put those into the dependencies, don't have the time right now to double check that the package names are 100% correct on the other distros. Will do so later (or when someone complains).